app/components/charts-generic/areas/areas-state.tsx

```diff
@@ -56,6 +71,9 @@ const useAreasState = ({
 
   const width = useWidth();
 
+  const hasSegment =
+    fields.segment && fields.segment.componentIri !== FIELD_VALUE_NONE;
```

Comment:
See comment above: is there a point in having a `segment` field with value none?